test/unit/dissemination-test.js

```diff
@@ -70,3 +70,80 @@ testRingpop('avoids redundant dissemination by filtering changes from source', f
         membership.checksum);
     assert.ok(changes.length > 0, 'changes issued');
 });
+
+testRingpop('raise piggyback counter on issueAsReceiver', function t(deps, assert) {
+    var ringpop = deps.ringpop;
+    var membership = deps.membership;
+    var dissemination = deps.dissemination;
+
+    var addrAlive = '127.0.0.1:3001';
+    var addrSuspect = '127.0.0.1:3002';
+    var addrFaulty = '127.0.0.1:3003';
+    var incNo = Date.now();
+
+    // Clear changes to start fresh, otherwise local member changes
+    // recorded during bootstrap phase would have been issued.
+    dissemination.clearChanges();
+
+    membership.makeAlive(addrAlive, incNo);
+    membership.makeSuspect(addrSuspect, incNo);
+    membership.makeFaulty(addrFaulty, incNo);
+
+    // 'sender' and source of updates are different; issues changes.
+    var changes;
+
+    assert.equal(dissemination.maxPiggybackCount, 15, 'assert maxPiggybackCount is 15');
+    for (var i = 0; i < dissemination.maxPiggybackCount;  i++) {
+        changes = dissemination.issueAsReceiver(addrAlive, incNo, membership.checksum);
+        assert.equal(changes.length, 3, 'changes issued');
```

Comment:
There's an implicit assumption being made here that the cluster / membership size is 3. To make this a bit more readable, can you do the following?

```
...
membership.makeAlive(...);
membership.makeSuspect(...);
membership.makeFaulty(...);
var expectedNumberOfChanges = membership.getMemberCount();

// then later...
assert.equal(changes.length, expectedNumberOfChanges);
```

Does that make sense?